packages/ecs-agent/ecs-agent.spec

```
@@ -86,6 +88,21 @@ Requires: %{_cross_os}amazon-ssm-agent
 %description
 %{summary}.
 
+%package base
```

nit: since `ecs-agent-config` is a more "obvious" name than `ecs-agent-base`:
```suggestion
%package config
```